src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -72,6 +72,9 @@ def register_options(cls, register):
     register('--delete-scratch', default=True, action='store_true',
              help='Leave intermediate scratch files around, for debugging build problems.')
 
+    register('--suppress', default=False, action='store_true',
+             help='Suppress jvm compiler output and warnings')
```

Comment:
... unless their is a failure.  Maybe "Suppress non-fatal jvm compiler compiler output..."